requirements/prod.in

```diff
@@ -10,6 +10,7 @@ jinja2<3
 keyring
 lxml
 MarkupSafe<2
+psutil
```

Comment:
Is there a 3pp record for this?

Reply:
https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qAH00000001fDYAQ/view